cmd/pocket/main.go

```diff
@@ -142,6 +149,32 @@ func commandArchive(arguments map[string]interface{}, client *api.Client) {
 	}
 }
 
+func commandAdd(arguments map[string]interface{}, client *api.Client) {
+	options := &api.AddOption{}
+
+	url, ok := arguments["<url>"].(string)
+	if !ok {
+		panic("Wrong arguments")
+	}
+
+	options.Url = url
+
+	if title, ok := arguments["--title"].(string); ok {
+		options.Title = title
+	}
+
+	if tags, ok := arguments["--tags"].(string); ok {
+		options.Tags = tags
+	}
+
+	err := client.Add(options)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Successfully added.")
```

Comment:
Sure, you can change that and merge! The rest of this file uses `panic`, do you want to change those as well? (I'm new to golang so I don't really know the difference)